src/applications/SRAnonTool/CTP_DicomToText.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+# Convert a DICOM file into plain text in a suitable format
+# for passing into the anonymisation part of SemEHR.
+# Usage: -y default.yaml -i input -o output
+#  -y = path to the default.yaml file to get FileSystemRoot and Mongo
+#  -i = input DICOM file, full path or relative to FileSystemRoot,
+#       or if not found then looked up in the MongoDB.
+#  -o = output filename or directory for the plain text file
+# Needs both dataLoad and dataExtract yaml files because Mongo is
+# defined in the former and the rest in the latter.
+
+import argparse
+import logging
+import os
+import sys
+import json
+import yaml
+import pydicom
+import xml.etree.ElementTree    # untangle and xmltodict not available in NSH
+from deepmerge import Merger    # for deep merging dictionaries
+sys.path.append('../../common/') # if we are in the application directory
+sys.path.append('src/common')    # if we are in the root of the repo
+if 'SMI_ROOT' in os.environ:     # $SMI_ROOT/lib
+    sys.path.append(os.path.join(os.environ['SMI_ROOT'], 'lib', 'python3'))
+sys.path.append('/home/arb/src/SmiServices/src/common')
```

Comment:
Do we need to rely on the contents of a user's home directory like this?

Reply:
We don't rely on it, but you're right it can be removed for deployment :)